src/Symfony/Component/String/AbstractString.php

```diff
@@ -254,6 +254,14 @@ public function collapseWhitespace(): self
         return $str;
     }
 
+    /**
+     * @param string|string[] $needle
+     */
+    public function containsAny($needle): bool
```

Comment:
```suggestion
    public function containsAny(string ...$needle): bool
```

Reply:
We considered using variadics but decided against in the rest of the API. So we should keep consistency here and use the polymorphic API instead.